packages/suite/src/components/suite/index.tsx

```diff
@@ -14,7 +14,7 @@ import WordInputAdvanced from './WordInputAdvanced';
 import Loading from './Loading';
 import BundleLoader from './BundleLoader';
 import FiatValue from './FiatValue';
-import WebusbButton from './WebusbButton';
+import { WebUsbButton } from './WebUsbButton';
```

Comment:
You have to rename the file too, right? Now the build is failing https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/2586622362

Reply:
On it. However, I do not have any problems locally 🤯

Reply:
<img width="621" alt="Screenshot 2022-06-20 at 9 18 58" src="https://user-images.githubusercontent.com/33235762/174546279-bfc49a93-be7c-4df1-af6d-f497dcd98a0a.png">
<img width="663" alt="Screenshot 2022-06-20 at 9 19 14" src="https://user-images.githubusercontent.com/33235762/174546315-2f7d6af9-c666-4856-b078-73427ef7fced.png">

:(

Reply:
Done, renamed to `WebUsbButtonTemp` in first commit and renamed to correct name in second commit.

Reply:
git configured to ignore case... you could try:
```
git mv packages/suite/src/components/suite/WebusbButton/ WebUsbButton
git mv WebUsbButton packages/suite/src/components/suite/WebUsbButton/ 

// commit change
```